net-mgmt/net-snmp/src/opnsense/mvc/app/models/OPNsense/Netsnmp/General.xml

```diff
@@ -31,6 +31,10 @@
             <default>0</default>
             <Required>Y</Required>
         </enableagentx>
+        <enableobservium type="BooleanField">
```

Comment:
same here, model bump for new default to render

Reply:
Done.